rest-server/core/src/main/scala/com/actionml/core/search/elasticsearch/ElasticSearchSupport.scala

```diff
@@ -88,6 +88,20 @@ class ElasticSearchClient private (alias: String, client: RestClient)(implicit w
   import ElasticSearchClient.ESVersions._
   import ElasticSearchClient._
   implicit val _ = DefaultFormats
+  private val esVersion: ESVersions.Value = {
+    import ESVersions._
+    Try {
+      val response = client.performRequest(new Request("GET", "/"))
+      val version = (JsonMethods.parse(response.getEntity.getContent) \ "version" \ "number").as[String]
+      if (version.startsWith("5.")) v5
+      else if (version.startsWith("6.")) v6
+      else if (version.startsWith("7.")) v7
+      else {
```

Comment:
@andreyshikov actually I think we can assume v8+ will work untill we find otherwise.